composer.json

```diff
@@ -11,16 +11,26 @@
         {
             "type": "git",
             "url" : "git@github.com:facebookincubator/facebook-for-woocommerce.git"
+        },
+        {
+            "type": "vcs",
+            "url": "https://github.com/woocommerce/action-scheduler-job-framework"
```

Comment:
Is this versioned - how do we know what version of the code we're getting? Does this get whatever is latest at time of `composer install`?

Reply:
Yeah if we want to go ahead with this separate job framework approach we should definitely tag a version of it before we merge this PR. Right now it's pulling from the `trunk` branch. I wanted to get a consensus about whether this is something we are willing to pursue as a squad. 

The framework is fairly light. It only contains the features we need for the FB extension so it could easily become part of this repo. However, the advantage of a separate framework is any other extension can use it. E.g. GLA, AutomateWoo, Pinterest...

Reply:
Sounds good - probably worth a discussion (P2, feedback from Action Scheduler maintainers). If other projects/teams are interested, probably good to start with the separate repo. I'm happy with either option👌 

I guess there's a third option - add it to Action Scheduler directly. (Maybe that's longer-term goal if it proves useful?)

Reply:
A P2 sounds good, will post...

> I guess there's a third option - add it to Action Scheduler directly

Potentially yes but it's very difficult to get anything added to AS. They're seem to be only accepting bug fixes for the last several months.